include/boost/multiprecision/detail/default_ops.hpp

```diff
@@ -2616,14 +2619,29 @@ inline BOOST_MP_CXX14_CONSTEXPR typename multiprecision::detail::expression<tag,
 template <class Backend, multiprecision::expression_template_option ExpressionTemplates>
 inline BOOST_MP_CXX14_CONSTEXPR multiprecision::number<Backend, ExpressionTemplates> erf BOOST_PREVENT_MACRO_SUBSTITUTION(const multiprecision::number<Backend, ExpressionTemplates>& arg)
 {
+   const int fpc_arg { (boost::multiprecision::fpclassify)(arg) };
```

Comment:
Using fpclassify here is a bit heavyweight just to test for zero, using `arg.is_zero()` would be better, but actually, why do we need this at all - surely boost::math::erf should handle this already?

Reply:
Resolved above